cmd/queue/main.go

```diff
@@ -197,7 +202,32 @@ func proxyHandler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, tracingEn
 	}
 }
 
-func knativeProbeHandler(healthState *health.State, prober func() bool, isAggressive bool, tracingEnabled bool, next http.Handler) http.HandlerFunc {
+func preferPodForScaledown(logger *zap.SugaredLogger, downwardAPILabelsPath string) bool {
+	// short circuit a rejection when no label path is set
+	if downwardAPILabelsPath == "" {
```

Comment:
So when will this ever be empty? Seems like the fundamental detection is missing whether or not the thing is mounted or not?